MatrixSDK/Data/MXSendReplyEventDefaultStringLocalizer.m

```diff
@@ -44,7 +44,7 @@ - (instancetype)init
         _senderSentTheirLocation = @"has shared their location.";
         _senderSentTheirLiveLocation = @"Live location.";
         _messageToReplyToPrefix = @"In reply to";
-        _replyToEndedPoll = @"Ended poll";
+        _endedPollMessage = @"Ended poll";
```

Comment:
These strings do not live in the regular localized strings file?

Reply:
Yes, generally clients can override them if they wish.
In this context they are meant to be **not** localized since we don't want to localize stuff from the sender side when sending an event.